node/pkg/watchers/evm/connectors/finalizer_poller.go

```diff
@@ -74,9 +76,8 @@ func (b *FinalizerPollConnector) SubscribeForBlocks(ctx context.Context, errC ch
 }
 
 func (b *FinalizerPollConnector) runFromSupervisor(ctx context.Context) error {
-	logger := supervisor.Logger(ctx).With(zap.String("eth_network", b.Connector.NetworkName()))
 	supervisor.Signal(ctx, supervisor.SignalHealthy)
-	return b.run(ctx, logger)
+	return b.run(ctx, b.logger)
 }
 
 func (b *FinalizerPollConnector) run(ctx context.Context, logger *zap.Logger) error {
```

Comment:
if `FinalizerPollConnector` has the field `logger`, is there a reason why we are passing the logger into the method instead of calling `b.logger` from the receiver?

```go
func (b *FinalizerPollConnector) methodA() {
  b.logger.Info("logging...")
 }
 
// instead of 
func (b *FinalizerPollConnector) methodA(logger *zap.Logger) {
  logger.Info("logging...")
 }
```

Reply:
seconded, would be good to clean up if it isn't needed anymore.